src/patchelf.cc

```diff
@@ -664,14 +665,16 @@ template<ElfFileParams>
 void ElfFile<ElfFileParamNames>::writeReplacedSections(Elf_Off & curOff,
     Elf_Addr startAddr, Elf_Off startOffset)
 {
-    /* Overwrite the old section contents with 'Z's.  Do this
-       *before* writing the new section contents (below) to prevent
-       clobbering previously written new section contents. */
-    for (auto & i : replacedSections) {
-        const std::string & sectionName = i.first;
-        const Elf_Shdr & shdr = findSectionHeader(sectionName);
-        if (rdi(shdr.sh_type) != SHT_NOBITS)
-            memset(fileContents->data() + rdi(shdr.sh_offset), 'Z', rdi(shdr.sh_size));
+    if (clobberOldSections) {
+        /* Overwrite the old section contents with 'Z's.  Do this
+        *before* writing the new section contents (below) to prevent
+        clobbering previously written new section contents. */
```

Comment:
Nit: didn't indentation of the comment get broken?  (it changed and personally I don't like it much)

Reply:
Fixed.